### PR TITLE
Feature/Karma-protocol-integrations

### PR DESCRIPTION
**Pull request by Alphonce-Mutebi**
- Karma protocol methods

### CODE REVIEW

File: src/utils/celo-integration/ContractMethods.js [hunk 1]
```diff
@@ -97,6 +101,40 @@ function ContractMethods(magic) {
...
+    * @param amount The amount used to calculate how much should be added or removed from
+                    a user's karma value. If the amount is positive, it will increase the 
+                    user's karma. If it is negative, it will decrease it. 
+    * @param updateFunctionKey An integer specifying which function should be used to update
+                               the user's karma. Setting the value to 1 will lead to a weighted 
+                               sum updated and setting it to 2 will lead to an averaged sum. 
+                               See README for details. => pass 2 for our case
+
+    **/
+    this.updateKarma = async (address, amount, updateFunctionKey) => {
```
Comment (AbijahKaj) [inline src/utils/celo-integration/ContractMethods.js:116]: You have to use the karmaContract variable you just created

File: src/utils/celo-integration/ContractMethods.js [hunk 2]
```diff
@@ -97,6 +101,40 @@ function ContractMethods(magic) {
...
+        return receipt
+    }
+
+    /**
+    * @dev Function to retireve the karma value for a user. 
+    * @param address The address of the user whose karma is being accessed
+    **/
+
+    this.getKarma = async (address) => {
+        let txObject = await contract.methods.getKarma(address);
```
Comment (AbijahKaj) [inline src/utils/celo-integration/ContractMethods.js:130]: Same thing, the Wakala Contract doesn't have that method, so it won't work.

**Review by AbijahKaj — CHANGES_REQUESTED** [review]

File: src/utils/celo-integration/ContractMethods.js (hunk 2 above)
Comment (Alphonce-Mutebi) [reply src/utils/celo-integration/ContractMethods.js]: Good Catch! I made the changes

**Review by AbijahKaj — APPROVED** [review]

File: src/utils/celo-integration/ContractMethods.js (hunk 1 above)
Comment (steffenkrogmann) [reply src/utils/celo-integration/ContractMethods.js:117]: great review!

**Review by sebastiaohns — APPROVED** [review]

**Pull request merged**